### DISCUSSION TITLE
Installing glibc from Brew

### DISCUSSION THREAD

**Discussion by janderssonse** [Homebrew/discussions#2253] [Linux] [answered]
CUT from a discussion  " All of our bottles currently require Glibc 2.23 or newer, and we install our own glibc if the host one is too old."-

But, if the host has let's say 2.24 but I still want to use 2.23 from the brew install (because I want to avoid dependency on the host, maybe i'm running gcc and binutils from brew also), is there any way to force the install of 2.23 with brew anyway in this situation. Right now one will only get a "Installing a version of glibc that is older than your system's can break formulae installed from source."

**Comment by carlocab** [comment, marked as answer]
You could get rid of the `depends_on BrewedGlibcNotOlderRequirement` from the formula to stop `brew` from preventing you from installing it.

Note, however, that there is no reason to do this: GCC and Binutils should work with the host Glibc. If you want to use this machine to build redistributable binary packages, the better thing to do would be to use Homebrew's Ubuntu 16.04 Docker container.

**Reply by janderssonse** [reply]
Not the answer I was hoping for :) but still an answer, thanks for taking the time to reply.
